library/sanitize.inc.php

```diff
@@ -77,3 +77,17 @@ function isWhiteFile($file)
 
     return false;
 }
+
+// check if a the first parameter contains a non-numeric value
```

Comment:
To clarify, this is:
Sanitize a value to ensure it is a number. Mostly utilized to prevent xss vulnerabilties.

Reply:
ok I will change that, thanks